packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -157,15 +158,17 @@ export class Meter implements api.Meter {
   }
 
   /**
-   * Creates a new value observer metric.
+   * Creates a new `ValueObserver` metric.
    * @param name the name of the metric.
    * @param [options] the metric options.
    * @param [callback] the value observer callback
    */
   createValueObserver(
     name: string,
     options: api.MetricOptions = {},
-    callback?: (observerResult: api.ObserverResult) => void
+    callback?: (
+      observerResult: api.ObserverResult
+    ) => Promise<unknown> | unknown
```

Comment:
Isn't this the same as

```suggestion
    ) => unknown
```